modules/pop3/pop3.go

```diff
@@ -17,15 +23,28 @@ type Connection struct {
 	Conn net.Conn
 }
 
+// Verifies that a POP3 banner begins with a valid status indicator
+func VerifyPOP3Contents(n int, ret []byte) (string, error) {
+	s := string(ret[0:n])
+	if strings.HasPrefix(s, "+OK "){
+			return s, nil
+	}
+	if strings.HasPrefix(s, "+ERR "){
+			return s, zgrab2.NewScanError(zgrab2.SCAN_APPLICATION_ERROR, errors.New("POP3 Reported Error"))
```

Comment:
Similar comment as for imap; this is actually a success case

Reply:
Based on the contents of the modbus scanner, it seems that SCAN_APPLICATION_ERROR _is_ [a success case](https://github.com/zmap/zgrab2/blob/11425ac78b64e39d95050b042c9505fa28c24641/modules/modbus/scanner.go#L140) - indicating that a valid response was received but the server reported back an error in the final ScanResult

Reply:
Though I just realized that string should be "-ERR" regardless

Reply:
`SCAN_APPLICATION_ERROR` can be treated as a "success" error case; the real indication is if the scanner returns `nil` for the results or not.  So it would be OK in this and the IMAP module to return `SCAN_APPLICATION_ERROR` here, except that in both of them `scanner.go` will omit results.

For example, from `pop3/scanner.go`:
```go
    banner, err := conn.ReadResponse()
    if err != nil {
        return zgrab2.TryGetScanStatus(err), nil, err
    }
```